go-controller/pkg/node/gateway.go

```diff
@@ -229,6 +229,18 @@ func gatewayInitInternal(nodeName, gwIntf string, subnets []*net.IPNet, gwNextHo
 		return bridgeName, uplinkName, nil, nil, err
 	}
 
+	chkPktLengthSupported, err := util.DetectCheckPktLengthSupport(bridgeName)
```

Comment:
actually should only do this check if config.Gateway.DisablePacketMTUCheck is not already set to true